easybuild/easyconfigs/a/AEDT/AEDT-2024R1.eb

```diff
@@ -0,0 +1,24 @@
+name = 'AEDT'
+version = '2024R1'
+local_short_ver = '241'
+local_bin_dir = '%(installdir)s/' + 'v%s/Linux64' % local_short_ver
+
+homepage = 'https://www.ansys.com/products/electronics'
+description = """The Ansys Electronics Desktop (AEDT) is a platform that enables
+true electronics system design. AEDT provides access to the Ansys gold-standard
+electromagnetics simulation solutions such as Ansys HFSS, Ansys Maxwell, Ansys Q3D
+Extractor, Ansys SIwave, and Ansys Icepak using electrical CAD (ECAD) and mechanical
+CAD (MCAD) workflows."""
+
+toolchain = SYSTEM
+
+sources = ['ELECTRONICS_%(version)s_LINX64.tgz']
+checksums = ['7b131adf981ebca1e2f4fe8e607e50323167b69e77180a0ab61b2759d57abca5']
+
+postinstallcmds = [
+    "echo '' > %s/VerifyOS.bash" % local_bin_dir,  # disable OS check
```

Comment:
I suggest to move these to the easyblock given that it is static code not depending the contents of the easyconfig.